pymongo/collection.py

```diff
@@ -686,7 +686,7 @@ def insert_many(self, documents, ordered=True,
 
         .. versionadded:: 3.0
         """
-        if not isinstance(documents, abc.Iterable) or not documents:
+        if not isinstance(documents, abc.Iterable) or isinstance(documents, abc.Mapping) or not documents:
```

Comment:
Can we make sure this to 80 characters or less?

Reply:
done